game-of-life-native/Game of Life/GameOfLifeViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ViewController.swift
+//  Game of Life
+//
+//  Created by Tobias Heine on 06.02.18.
+//  Copyright © 2018 Novoda. All rights reserved.
+//
+
+import UIKit
+import Foundation
+import KotlinGameOfLife
+
+class GameOfLifeViewController: UIViewController, KGOLAppView {
```

Comment:
`KGOLAppView` see [here](https://github.com/novoda/spikes/pull/364/files#diff-b6035d0296c22e53a985add47c90a9e8) the kotlin implementation.